src/js/files/api.js

```diff
@@ -19,10 +19,12 @@ export const find = ({ fileType }) =>
  * @param page {number} the page of results to get
  * @returns {promise}
  */
-export const list = ({ fileType }) =>
+export const list = ({ fileType, page }) =>
     Request.get("/api/uploads").query({
         type: fileType,
```

Comment:
```suggestion
        upload_type: fileType,
```
Not your change, but api expects `upload_type` rather than `type`. Filtering for the correct type should work again with this change